core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -232,7 +232,7 @@ object LogConfig {
     }
   }
 
-  private val configDef: LogConfigDef = {
+  val configDef: LogConfigDef = {
```

Comment:
Actually, I need to access to the `LogConfig.configDef.serverDefaultConfigNames` from an internal project we've at LinkedIn. That's how I discovered the problem. The tests needs it too.